src/script/components/calling/GroupVideoGridTile.tsx

```diff
@@ -64,13 +66,24 @@ const GroupVideoGridTile: React.FC<GroupVideoGridTileProps> = ({
   const activelySpeakingBoxShadow = `inset 0px 0px 0px 1px var(--group-video-bg), inset 0px 0px 0px 4px var(--accent-color), inset 0px 0px 0px 7px var(--app-bg-secondary)`;
   const groupVideoBoxShadow = participantCount > 1 ? 'inset 0px 0px 0px 2px var(--group-video-bg)' : 'initial';
 
+  const onTileClick = () => onParticipantDoubleClick(participant?.user.qualifiedId, participant?.clientId);
+
+  const onTileEnterClick = (keyboardEvent: React.KeyboardEvent) => {
+    if (!isEnterKey(keyboardEvent)) {
+      return;
+    }
+
+    onParticipantDoubleClick(participant?.user.qualifiedId, participant?.clientId);
```

Comment:
You could use the `onTitleClick` here (since it's already bound to the correct data). 
And let's maybe rename it `handleTitleClick` to name it like an action